rpc.c

```diff
@@ -758,15 +758,41 @@ _rpc_msg_decode_tree (rpc_message msg, GNode *root)
 
                     /* Actually create the root node. */
                     root = APTERYX_NODE (NULL, path);
+
+                    /* Add null value - this may be superceded later */
+                    g_node_prepend_data(root, NULL);
```

Comment:
Is there a cleaner way of doing this - its a bit ugly to have to remove the node below.